server/lib/src/value.rs

```diff
@@ -774,6 +776,25 @@ pub struct Session {
     pub scope: SessionScope,
 }
 
+impl fmt::Debug for Session {
+    fn fmt(&self, f: &mut Formatter) -> fmt::Result {
+        let issuer = match self.issued_by {
+            IdentityId::User(u) => uuid_to_proto_string(u),
+            IdentityId::Synch(u) => uuid_to_proto_string(u),
+            _ => "invalid".to_string(),
```

Comment:
It's not invalid - match the remaining cases, because Internal is a valid issuer!

Reply:
Wopsies my bad there, I meant internal instead of invalid :p 
I should have fixed everything now! 
And also thanks for all the helpful feedback :)

Reply:
And please don't `_`-match an enum - explicitly match the `Internal` variant.

Reply:
Yeah, it's always better to explicitly match the variants @Seba-T because that way if we add a new variant or change things, the complier will flag we now need to handle the cas differently in this code path. It's always better to be explicit, there are very few cases where I'll use a '_' match because we need to be thorough.

Reply:
Alright, thanks for the tip!